lib/src/Pos/Launcher/Scenario.hs

```diff
@@ -98,13 +97,6 @@ runNode' NodeResources {..} workers' plugins' = ActionSpec $ \vI sendActions ->
         firstGenesisHash
         (gdFtsSeed genesisData)
 
-    lastKnownEpoch <- LrcDB.getEpoch
```

Comment:
Why don't we print leaders on start now?

Reply:
Read vatnik.